tests/test_integrations.py

```diff
@@ -477,8 +511,13 @@ def _echo(args):
 @pytest.mark.parametrize("case", ALL_PLATFORMS)
 def test_script(case):
     script, exp_out, exp_rtn = case
+    if ON_WINDOWS and 'skip_if_on_windows' in script:
```

Comment:
Use `pytest.param` with `skip_if_on_windows` instead

Reply:
Please take a look to the `test_integrations.py`. It has a different structure than other test files.